src/Google-Plus/Provider.php

```diff
@@ -66,10 +66,11 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'     => $user['id'], 'nickname' => Arr::get($user, 'nickname'),
-            'name'   => $user['displayName'],
-            'email'  => $user['emails'][0]['value'],
-            'avatar' => Arr::get($user, 'image.url'),
+            'id' => $user['names'][0]['metadata']['source']['id'],
+            'nickname' => $user['names'][0]['displayName'],
```

Comment:
Are all of this values always set?

Reply:
Not found info about it. Due to this add some checks via Arr:get

Docs and test tool
https://developers.google.com/people/api/rest/v1/people/get?apix_params=%7B%22resourceName%22%3A%22people%2Fme%22%2C%22personFields%22%3A%22photos%22%7D